pkg/util/ip_address.go

```diff
@@ -82,5 +82,8 @@ func SplitHostPortDefault(input, defaultHost, defaultPort string) (NetworkAddres
 
 // SplitHostPort splits ip address/hostname string by host and port
 func SplitHostPort(input string) (NetworkAddress, error) {
+	if len(input) == 0 {
```

Comment:
Otherwise [this](https://github.com/grafana/grafana/blob/master/pkg/util/ip_address_test.go#L38) test fails.

Reply:
Make sense :+1: